<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Knapp Apprentice Training</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Angemeldet:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="5">
     <widget class="QLabel" name="lab_INFO">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_eingeloggt_als">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>37</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" rowspan="2" colspan="5">
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset>
         <normaloff>../../_Symbole/Icons/misc/data.ico</normaloff>../../_Symbole/Icons/misc/data.ico</iconset>
       </attribute>
       <attribute name="title">
        <string>Stammdaten</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton_lehrlinge">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Lehrlinge</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../_Symbole/Icons/misc/MISC28.ICO</normaloff>../../_Symbole/Icons/misc/MISC28.ICO</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_ausbilder">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Ausbilder</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../_Symbole/Icons/misc/trainer.ICO</normaloff>../../_Symbole/Icons/misc/trainer.ICO</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButton_Unterrichtsthemen">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Unterrichtsthemen</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../_Symbole/UsedSymbols/coordinated_assessment_documents_report_icon_148377.ico</normaloff>../../_Symbole/UsedSymbols/coordinated_assessment_documents_report_icon_148377.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_raeume">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Räume</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../_Symbole/Icons/misc/place.ico</normaloff>../../_Symbole/Icons/misc/place.ico</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_lehrberufe">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Fachrichtungen</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../_Symbole/Icons/misc/training.ico</normaloff>../../_Symbole/Icons/misc/training.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_spezialmodule">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Spezialmodule</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../_Symbole/Icons/misc/folder.ico</normaloff>../../_Symbole/Icons/misc/folder.ico</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButton_LAP_Themenkatalog">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>LAP-Themenkatalog</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../_Symbole/UsedSymbols/2020-12-01 14_02_39-WIKA_OO1.jpg (590×182).ico</normaloff>../../_Symbole/UsedSymbols/2020-12-01 14_02_39-WIKA_OO1.jpg (590×182).ico</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_lehrstoff">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Lehrstoff</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../_Symbole/Icons/misc/MISC100.ICO</normaloff>../../_Symbole/Icons/misc/MISC100.ICO</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="icon">
        <iconset>
         <normaloff>../../_Symbole/Icons/misc/share_analytics.ico</normaloff>../../_Symbole/Icons/misc/share_analytics.ico</iconset>
       </attribute>
       <attribute name="title">
        <string>Bewegungsdaten</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>571</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_Wochen_U">
             <property name="text">
              <string>Wochen-Unterricht</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../_Symbole/UsedSymbols/education_cap_icon_157723.ico</normaloff>../../_Symbole/UsedSymbols/education_cap_icon_157723.ico</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="pushButton_Berufsschulzeiten">
             <property name="text">
              <string>Berufsschulzeiten</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../_Symbole/UsedSymbols/school_bell_icon_175715.ico</normaloff>../../_Symbole/UsedSymbols/school_bell_icon_175715.ico</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="icon">
        <iconset>
         <normaloff>../../_Symbole/Icons/misc/assessment.ico</normaloff>../../_Symbole/Icons/misc/assessment.ico</iconset>
       </attribute>
       <attribute name="title">
        <string>Auswertung</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="6" column="0" colspan="5">
     <widget class="QPushButton" name="pushButton_abmelden">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Abmelden</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../_Symbole/Icons/computer/log_out.ico</normaloff>../../_Symbole/Icons/computer/log_out.ico</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../_Symbole/UsedSymbols/Knapp_Logo.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>KAT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
